website/oauth/models/__init__.py

```diff
@@ -489,7 +491,7 @@ class ApiOAuth2Application(StoredObject):
     name = fields.StringField(index=True, required=True, validate=[string_required, MaxLengthValidator(200)])
     description = fields.StringField(required=False, validate=MaxLengthValidator(1000))
 
-    date_created = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow,
+    date_created = fields.DateTimeField(auto_now_add=timezone.now,
```

Comment:
psql doesn't enforce it but the django ORM will. You'll get an exception if you try and set it to anything, ever.